app/services/payments/update_payment_method_data_service.rb

```diff
@@ -4,15 +4,21 @@ module Payments
   class UpdatePaymentMethodDataService < BaseService
     Result = BaseResult[:payment]
 
-    def initialize(payment:, provider_payment_method_id:)
+    def initialize(payment:, provider_payment_method_id:, force: false)
```

Comment:
Not sure to understand this `force` argument. When are we planning to call it with `force: true`?

Reply:
Yeah I hesitated with this one.

I feel weird calling `UpdateThisThing` and the service doesn't update it. 
Originally, I had put the check in the job:

```ruby 
def perform(...)
  if payment.provider_payment_method_id != provider_payment_method_id
    ::Payments::UpdatePaymentMethodDataService.call!(payment:, provider_payment_method_id:)
  end
end
```

I thought I'd put in the service but then the service is weird. I liked the force option but maybe a better alternative is to rename the job:

* `SetPaymentMethodDataService`
* `AddPaymentMethodDataService`
* `CreatePaymentMethodDataService`
* ?

What do you think?

Reply:
removed force and renamed!